docs/api-usage/authentication.mdx

```diff
@@ -133,6 +192,46 @@ mutation {
 }
 ```
 
+## Typical authentication flow for a storefront
+
+In the most common scenario, you'll use the `tokenCreate` mutation with user's email and password when they try logging in inside a storefront.
+
+You'll receive an access token and a refresh token. Depending on your use-case, you might consider **storing refresh token** in `localStorage` so that the token is persisted between page reloads and browser sessions.
```

Comment:
```suggestion
The mutation will return an access token and a refresh token. Depending on your use case, you might consider **storing refresh token** in `localStorage` so that the token is persisted between page reloads and browser sessions.
```